### ISSUE TITLE
one repo builds packages and one fails

### ISSUE THREAD

**Issue by JimT787** [shannah/jdeploy#94]
Wondering if there is a simple answer.

I am trying to use private repositories that publish releases to public repository.

I have a test app that did fine and another that is just failing for some reason.  I have followed the same process in each application with the github actions file.  Have Github tokens as secrets stored in the private repository.

I included screenshots of the build process.  Both apps build, but the jdeploy process has two different results.  Seems to be related to to "jdeploy@4.0.8"

One build process doesn't see it so provides a Warning, then downloads it and continues.  The other build process doesn't see it and quits instead of downloading and installing it.

Thanks for your help!

![Failed](https://user-images.githubusercontent.com/98830368/218360783-1159c9f7-7be3-4f5a-b8c7-6fbaa666597c.png)
![Passed](https://user-images.githubusercontent.com/98830368/218360788-a7456d42-06be-4827-b096-7727d00d2bed.png)

**Comment by shannah** [comment]
Does the repo have a package.json file?

**Comment by JimT787** [comment]
It did not--I was down the path of Github versus NPM and for some reason thought I didn't need it if I did it that way--brain is just drained after many days of working on this.

Thank you for your quick reply!

I added the package.json and it worked as advertised!  I am very happy to have it working.  It saves me getting another computer and I signed up for your substack to toss you some bucks.  Really appreciate the effort you put into this--it looks like it will work out very nicely for me and my users.  It's just a hobby level program I do as a volunteer project and jDeploy will help out greatly with this project.
